plugins/node/opentelemetry-instrumentation-express/test/express.test.ts

```diff
@@ -289,22 +281,30 @@ describe('ExpressInstrumentation', () => {
 
   describe('Disabling plugin', () => {
     it('should not create new spans', async () => {
+      instrumentation.disable();
       const rootSpan = tracer.startSpan('rootSpan');
-      const app = express();
-      app.use(express.json());
-      app.use((req, res, next) => {
-        for (let i = 0; i < 1000; i++) {
-          continue;
+      const { server, port } = await serverWithMiddleware(
+        tracer,
+        rootSpan,
+        app => {
+          app.use(express.json());
+          const customMiddleware: express.RequestHandler = (req, res, next) => {
+            for (let i = 0; i < 1000; i++) {
+              continue;
+            }
+            return next();
+          };
+          app.use(customMiddleware);
```

Comment:
I believe we could further simplify this test by removing this.

Reply:
I think it's here to ensure that the instrumentation is also disabled on the app level, not only router level.
Treating Express as a black box keeping at least one of those middleware makes sense imho.